### PR TITLE
Add CollectionManager, consistent xmin,xmax settings in pad, update Run2 lumi

### PR DESCRIPTION
**Pull request by okepka**

### CODE REVIEW

File: src/plotter/presets.py [hunk 1]
```diff
@@ -135,7 +141,20 @@ def add_and_plot(
...
 
-    def remove_empty(self):
+        self.update_ranges()
+
+    #       self.ratioPad.update_range()
+
+    def update_ranges(self):
+
+        if self.nonEmpty and not self.custom_xrange:
+            (xmin, xmax) = self._xrange_emptysupressed()
```
Comment (fnechans) [inline src/plotter/presets.py:151]: Dont see how `xmin` and `xmax` is used? Shouldn't `self.mainPad.set_xrange(xmin, xmax)` be called or something?

File: src/plotter/pad.py [hunk 2]
```diff
@@ -311,6 +326,11 @@ def set_yrange(self, yMin: float = 0, yMax: float = 1) -> None:
             self._set_basis_yrange()
 
     def _set_basis_xrange(self) -> None:
+
+        # only for setting both limits at the time for now TODO
+        if not self.xMin and not self.xMax:
+            return
+
         """Sets rangeof the x-axis through the basis histogram"""
```
Comment (fnechans) [inline src/plotter/pad.py:334]: Doc string should be moved before the code.

**Review by fnechans — CHANGES_REQUESTED** [review]

File: src/plotter/presets.py [hunk 3]
```diff
@@ -135,7 +141,20 @@ def add_and_plot(
 
         self.ratioPad.plot_histos()
 
-    def remove_empty(self):
+        self.update_ranges()
+
+    #       self.ratioPad.update_range()
```
[the remark this answers is not in the document]
Comment (okepka) [reply src/plotter/presets.py:146]: removed

File: src/plotter/presets.py (hunk 1 above)
Comment (okepka) [reply src/plotter/presets.py:151]: yes, it got dropped after refactoring

**Comment by okepka** [comment]
@fnechans comments implemented

**Review by fnechans — APPROVED** [review]

**Pull request merged**